Vagrantfile

```diff
@@ -1,17 +1,11 @@
 Vagrant.configure("2") do |config|
   # The base OS 
-  config.vm.box = "ubuntu/trusty64"
+  config.vm.box = "ubuntu/bionic64"

```

Comment:
jammy?  https://wiki.ubuntu.com/Releases

Reply:
> jammy? https://wiki.ubuntu.com/Releases

TBH not sure what you want to do with Vagrant here in terms of legacy support - clearly there are a couple of LTS Ubuntu releases since bionic. Personally I'd go for 22.04.2 LTS (Jammy Jellyfish).

Reply:
have updated to jammy